Dockerfile

```
@@ -1,17 +1,16 @@
-FROM rust:1 AS chef 
-RUN cargo install cargo-chef 
-WORKDIR /app
+FROM rust:1 AS builder 
 
-FROM chef AS planner
-COPY . .
-RUN cargo chef prepare --recipe-path recipe.json
+WORKDIR /app
 
-FROM chef as builder
+COPY Cargo.lock Cargo.toml ./
+RUN mkdir src && echo "fn main() {}" > src/main.rs
+COPY build.rs .
+COPY proto/stationapi.proto proto/stationapi.proto
 RUN apt-get update && \
-    apt-get install -y protobuf-compiler libprotobuf-dev && \
+    apt-get install -y protobuf-compiler libprotobuf-dev rsync && \
     rm -rf /var/lib/apt/lists/*
-COPY --from=planner /app/recipe.json recipe.json
-RUN cargo chef cook --release --recipe-path recipe.json
+RUN cargo build --release
+
 COPY . .
 RUN SQLX_OFFLINE=true cargo build --release
```

[ask]
なんでcargo build2回必要なんだっけ？必要だとしても `SQLX_OFFLINE=true` がないとsqlxに怒られる気がするけどビルド通ってるしいらないのかな。
https://github.com/TrainLCD/StationAPI/pull/634/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R12

[nits]
cargo buildで標準出力使わなかったら若干速くなるのかな（別に対応しなくてもいいけど）
```suggestion
RUN SQLX_OFFLINE=true cargo build --release --quiet
```

---

IO遅いマシンで動いてるから標準出力吐かせなくするのは効果ありそうかも
見れるときに対応しまーす

---

[imo]
`RUN cargo build --release` は多分不要。依存関係だけキャッシュさせるという意図であっても今は依存関係がどうなるかは誰にもわからん開発が黎明期なので、一旦外して通しでどの程度キャッシュされて時間が短縮されるか測定した後適用して差分見るとかが良いと思ってます